README.md

```diff
@@ -1,366 +1,542 @@
+`<new_folder_name>` recursively, effectively renaming `<folder_to_upload>` to
+`<new_folder_name>` in the archive.
 
 ### Encrypt on upload
 
-It is possible to combine the encryption and upload steps with the use of the flag `--encrypt-with-key` followed by the path of the crypt4gh public key to be used for encryption. In this case, the input list of file arguments can only contain _unencrypted_ source files. For example the following,
+You can combine the encryption and upload steps using the `-encrypt-with-key`
+flag, followed by the path to the Crypt4GH public key for encryption. In this
+case, the input list of files can only contain unencrypted files. For
+example:
 
 ```bash
-./sda-cli -config <configuration_file> upload --encrypt-with-key <public_key> <unencrypted_file_to_upload>
+./sda-cli -config <configuration_file> upload -encrypt-with-key <public_key_file> <unencrypted_file_to_upload>
 ```
 
-will encrypt `<unencrypted_file_to_upload>` using `<public_key>` as public key and upload the created `<file_to_upload.c4gh>` in the base folder of the user.
+This command encrypts `<unencrypted_file_to_upload>` using `<public_key_file>`
+and uploads the resulting `<file_to_upload.c4gh>` to the user's base folder.
 
-Encrypt on upload can be combined with any of the flags above. For example,
+The encrypt on upload feature can be combined with other flags. For example:
 
 ```bash
-./sda-cli -config <configuration_file> upload --encrypt-with-key <public_key> -r <folder_to_upload_with_unencrypted_data> -targetDir <new_folder_name>
+./sda-cli -config <configuration_file> upload -encrypt-with-key <public_key_file> -r <folder_to_upload_with_unencrypted_data> -targetDir <new_folder_name>
 ```
 
-will first encrypt all files in `<folder_to_upload_with_unencrypted_data>` and then upload the folder recursively (selecting only the created `c4gh` files) under `<new_folder_name>`.
+This command encrypts all files in `<folder_with_unencrypted_data>` and uploads
+the folder recursively (including only the resulting `.c4gh` files) under
+`<new_folder_name>`.
 
-**Notes**: The tool calls the [encrypt](#Encrypt) module internally, therefore similar behavior to that command is expected, including the creation of hash files. In addition,
+**Notes**:
 
-- For encryption with [multiple public keys](<#Encrypt-file(s)-with-multiple-keys>), concatenate all public keys into one file and pass it as the argument to `encrypt-with-key`.
-- If the input includes encrypted files, the tool will exit without performing further tasks.
-- The encrypted files will be created next to their unencrypted counterparts.
-- The tool will not overwrite existing encrypted files. It will exit early if encrypted counterparts of the source files already exist with the same source path.
-- If the flag `--force-overwrite` is used, the tool will overwrite any already existing file.
-- The cli will exit if the input has any un-encrypred files. To override that, use the flag `--force-unencrypted`.
```

Comment:
Does the mention to `--force-unencrypted` need to be removed from here?

Reply:
Since this an option primarily for testing maybe it's good to move it under the Developer's section.

Reply:
Fixed in 9f60755